vital/video_metadata/convert_0104_metadata.cxx

```diff
@@ -40,9 +40,55 @@
 
 #include <vital/exceptions/klv.h>
 
+#include <vital/types/geodesy.h>
+
 namespace kwiver {
 namespace vital {
 
+namespace {
+
+// ----------------------------------------------------------------------------
+template < int N = 2 >
+Eigen::Matrix< double, N, 1 >
+empty_vector()
+{
+  Eigen::Matrix< double, N, 1 > v;
+  for ( int i = 0; i < N; ++i )
+  {
+    v[i] = std::numeric_limits<double>::quiet_NaN();
+  }
+
+  return v;
+}
+
+// ----------------------------------------------------------------------------
+template < int N = 2 >
+bool
+is_empty( Eigen::Matrix< double, N, 1 > const& vec )
+{
+  for ( int i = 0; i < N; ++i )
+  {
+    if ( std::isnan( vec[i] ) )
+    {
+      return true;
+    }
+  }
+
+  return false;
+}
+
+// ----------------------------------------------------------------------------
+bool
+is_valid_lat_lon( vector_2d const& vec )
```

Comment:
I blame English :smile:. (Or the mixed conventions to talk about "X, Y" and "Lat, Lon", in those orders, anyway.)

Should I just rename it `is_valid_lon_lat`?